### PR TITLE
MINOR: Refresh of the docs

### PR DESCRIPTION
**Pull request by mimaison**
Regenerated the docs from the 3.8 branch

### CODE REVIEW

File: 38/upgrade.html [hunk 1]
```diff
@@ -19,7 +19,7 @@
 
 <script id="upgrade-template" type="text/x-handlebars-template">
 
-<h4><a id="upgrade_3_8_0" href="#upgrade_3_8_0">Upgrading to 3.8.0 from any version 0.8.x through 3.7.x</a></h4>
+<h4><a id="upgrade_3_8_0" href="#upgrade_3_8_0">Upgrading to 3.8.0 from any version 0.8.x through 3.6.x</a></h4>
```
Comment (jlprat) [inline 38/upgrade.html]: This should be 3.7.x

File: 38/upgrade.html [hunk 2]
```diff
@@ -28,8 +28,7 @@ <h5><a id="upgrade_380_notable" href="#upgrade_380_notable">Notable changes in 3
             If READ is not authorized, checkpointing is limited to offsets mirrorred after the start of the task.
             See <a href="https://issues.apache.org/jira/browse/KAFKA-15905">KAFKA-15905</a> for more details.
         </li>
-        <li>JBOD in KRaft is no longer in early access.</li>
-        <li>Tiered Storage, which is still in early access, now supports clusters configured with multiple log directories (i.e. JBOD feature).</li>
+        <li>Tiered Storage now supports clusters configured with multiple log directories (i.e. JBOD feature).</li>
```
Comment (jlprat) [inline 38/upgrade.html]: This change should be reverted I believe

File: 38/upgrade.html [hunk 3]
```diff
@@ -28,8 +28,7 @@ <h5><a id="upgrade_380_notable" href="#upgrade_380_notable">Notable changes in 3
             If READ is not authorized, checkpointing is limited to offsets mirrorred after the start of the task.
             See <a href="https://issues.apache.org/jira/browse/KAFKA-15905">KAFKA-15905</a> for more details.
         </li>
-        <li>JBOD in KRaft is no longer in early access.</li>
```
Comment (jlprat) [inline 38/upgrade.html]: This should stay

**Review by jlprat — COMMENTED** [review]
I found 3 places where the PR you based this work on was working on an outdated doc page. I pointed them up in the comments.

**Comment by mimaison** [comment]
Thanks for the review. So it looks like some changes were only done in kafka-site and are not in 3.8. I pushed an update to fix these.

**Comment by jlprat** [comment]
I thought I ported both there, maybe I overlooked a couple, I'll check later if nobody does

**Review by jlprat — APPROVED** [review]
LGTM thanks @mimaison

**Comment by jlprat** [comment]
This was the PR for trunk https://github.com/apache/kafka/pull/16654

**Pull request merged**